istioctl/cmd/workload.go

```diff
@@ -240,6 +241,8 @@ Configure requires either the WorkloadGroup artifact path or its location on the
 	configureCmd.PersistentFlags().BoolVar(&autoRegister, "autoregister", false, "Creates a WorkloadEntry upon connection to istiod (if enabled in pilot).")
 	configureCmd.PersistentFlags().BoolVar(&dnsCapture, "capture-dns", true, "Enables the capture of outgoing DNS packets on port 53, redirecting to istio-agent")
 	configureCmd.PersistentFlags().StringVar(&workloadIP, "workloadIP", "", "IP address of the workload used in the WorkloadEntry")
+	configureCmd.PersistentFlags().StringVar(&outputCerts, "outputCerts", "/var/run/secrets/istio", "The output directory for the key and certificate. Must be set for VMs using provisioning certificates")
```

Comment:
I am not sure this is a good default. These folders are ephemeral and will not survive a restart

Reply:
Also we default it here I think: https://github.com/istio/istio/blob/971ab85c99f5080866390ce23e7e6ee8a7fba0e0/tools/packaging/common/istio-start.sh#L108

Reply:
Thanks for this info, Let's use `/etc/certs` as default to make the alignment.